webpack-common.js

```diff
@@ -0,0 +1,72 @@
+const ExtractTextPlugin = require('extract-text-webpack-plugin');
+
+const postCssPlugins = [];
+const urlLoaderOptions = {
+  limit: 10000,
+};
+const styleRules = [
+  {
+    test: /\.css$/,
+    loader: ExtractTextPlugin.extract({
+      fallback: 'style-loader',
+      use: [
+        {
+          loader: 'css-loader',
+          options: { importLoaders: 2, sourceMap: true },
+        },
+        {
+          loader: 'postcss-loader',
+          options: {
+            outputStyle: 'expanded',
+            plugins: () => postCssPlugins,
+            sourceMap: true,
+            sourceMapContents: true,
+          },
+        },
+      ],
+    }),
+  },
+  {
+    test: /\.scss$/,
+    loader: ExtractTextPlugin.extract({
+      fallback: 'style-loader',
+      use: [
+        {
+          loader: 'css-loader',
+          options: { importLoaders: 2, sourceMap: true },
+        },
+        {
+          loader: 'postcss-loader',
+          options: { plugins: () => postCssPlugins },
+        },
+        {
+          loader: 'sass-loader',
+          options: {
+            outputStyle: 'expanded',
+            sourceMap: true,
+            sourceMapContents: true,
+          },
+        },
+      ],
+    }),
+  },
```

Comment:
Actually all references to css and scss or sass files can and should be removed since we don't make use of them. So the whole `styleRules` can be removed.